controllers/userController.js

```diff
@@ -0,0 +1,39 @@
+const Sequelize = require("sequelize");
+const bcrypt = require("bcrypt");
+const sequelize = require("../util/database");
+const user = require("../models/user");
+
+const userRegister = async (req, res) => {
+  try {
+    const {
+      username,
+      password,
+      email,
+      firstName,
+      lastName,
+      roleId,
+      confirmPassword,
+    } = req.body;
+    if (password !== confirmPassword) {
+      return res
+        .status(402)
```

Comment:
@amrit56789 why status code 402?
402 code is for Payment Required??? Can you check status codes and send proper